Apps/Src/BlinkLights.c

```diff
@@ -8,7 +8,7 @@
 void Task_BlinkLight(void* p_arg){
     OS_ERR err;
     while(1){
-        Lights_MultiToggle();
+        
```

Comment:
Why not just delete this task if it does nothing?

Reply:
#199 will merge in the proper behavior.